Makefile.gh

```diff
@@ -32,6 +32,7 @@ bandit:
 
 
 propagate-version:
+	echo "$(VERSION)" > VERSION
```

Comment:
Could you please provide a bit of context in the commit message? Just for future readers?

Reply:
Hopefully it's clearer now